lib/gpu/context.cc

```diff
@@ -2,27 +2,37 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "flutter/lib/ui/gpu/context.h"
+#include "flutter/lib/gpu/context.h"
 
-#include <memory>
-#include <sstream>
+#include <future>
 
-#include "flutter/fml/log_level.h"
-#include "flutter/fml/logging.h"
-#include "flutter/fml/make_copyable.h"
-#include "flutter/fml/memory/ref_ptr.h"
+#include "dart_api.h"
 #include "flutter/lib/ui/ui_dart_state.h"
-#include "third_party/tonic/dart_wrappable.h"
+#include "fml/make_copyable.h"
+#include "tonic/converter/dart_converter.h"
 
 namespace flutter {
 
-IMPLEMENT_WRAPPERTYPEINFO(ui, GpuContext);
+IMPLEMENT_WRAPPERTYPEINFO(gpu, GpuContext);
 
-std::string GpuContext::InitializeDefault(Dart_Handle wrapper) {
-  auto dart_state = UIDartState::Current();
+GpuContext::GpuContext(std::shared_ptr<impeller::Context> context)
```

Comment:
Nit: Can the name of this class just be `Context`? Like the `impeller::` counterpart?

Reply:
Done.